ledger/sandbox/src/main/scala/com/digitalasset/platform/store/dao/LedgerDao.scala

```diff
@@ -32,13 +32,16 @@ import scala.concurrent.Future
 
 trait LedgerReadDao extends ReportsHealth {
 
+  /** The maximum number of concurrent connections */
```

Comment:
Probably just moving `concurrent` to the name makes more sense.

Reply:
Done.